scripts/scheduled-task-commands/disableOldUsers.php

```diff
@@ -0,0 +1,29 @@
+<?php
+require_once 'globals.php';
+require_once LIB_PATH . '/php-commons/Db.php';

```

Comment:
require_once getenv('APP_LIBS_PATH') . 'globals.php';
require_once getenv('APP_LIBS_PATH') . 'globals_plus.php';

These should be used now rather than the above. The db is already required in globals_plus.php

Reply:
If you can't use the globals_plus.php (this is to get the portal_path) then you can include the db with this

require_once getenv('APP_PATH') . '/Leaf/Db.php';